app/javascript/mastodon/features/emoji/emoji_mart_data_light.ts

```diff
@@ -0,0 +1,55 @@
+// The output of this module is designed to mimic emoji-mart's
+// "data" object, such that we can use it for a light version of emoji-mart's
+// emojiIndex.search functionality.
+import type { BaseEmoji } from 'emoji-mart';
+import type { Emoji } from 'emoji-mart/dist-es/utils/data';
+
+import type { Search, ShortCodesToEmojiData } from './emoji_compressed';
+import emojiCompressed from './emoji_compressed';
+import { unicodeToUnifiedName } from './unicode_to_unified_name';
+
+type Emojis = {
+  [key in keyof ShortCodesToEmojiData]: {
+    native: BaseEmoji['native'];
+    search: Search;
+    short_names: Emoji['short_names'];
+    unified: Emoji['unified'];
+  };
+};
+
+const [
+  shortCodesToEmojiData,
+  skins,
+  categories,
+  short_names,
+  emojisWithoutShortCodes, // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
If you prefix the name with `_`, like `_emojisWithoutShortCodes`, then you dont need to ignore the line

Reply:
Thanks, fixed it.